traitsui/testing/tester/default_registry.py

```diff
@@ -23,14 +23,13 @@ def get_default_registry():
     -------
     registry : TargetRegistry
         The default registry containing implementations for TraitsUI editors
-        that is toolkit specific.  
+        that is toolkit specific.
     """
-    # side-effect to determine current toolkit
-    from pyface.toolkit import toolkit_object
```

Comment:
I think we still need this here. Let's bring this back and tell flake8 not to complain about this line, see https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors